<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Baza danych SQL</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="mywindow.qrc" >:/images/qt.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Książki i autorzy</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="4" >
       <widget class="QTableView" name="tableView" />
      </item>
      <item row="1" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>341</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="pushButtonDeleteBook" >
        <property name="text" >
         <string>Usuń ksążkę</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QPushButton" name="pushButtonNewBook" >
        <property name="text" >
         <string>Nowa książka</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QPushButton" name="pushButtonNewAuthor" >
        <property name="text" >
         <string>Nowy autor</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>461</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="pushButtonExit" >
     <property name="text" >
      <string>Wyjście</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="mywindow.qrc" />
 </resources>
 <connections/>
</ui>
